ext/io/lib.rs

```diff
@@ -317,14 +333,29 @@ impl Resource for ChildStderrResource {
   }
 }
 
-#[derive(Clone, Copy)]
+#[cfg(windows)]
+use winapi::um::wincon::CONSOLE_SCREEN_BUFFER_INFO;
```

Comment:
Nitpick: It's probably just better to inline these if there's only one used rather than using conditional compilation for them.